app/root.go

```diff
@@ -35,5 +43,6 @@ var (
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&templateFilePath, "file", "f", "", "specify the path of a data-driven template")
 	RootCmd.PersistentFlags().StringVarP(&payload, "payload", "p", "", "specify a map of YAML files, TOML files, JSON files, .env files and raw data")
-	RootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "display a detailed output")
+	RootCmd.PersistentFlags().BoolVarP(&info, "info", "i", false, "set logging to info level")
```

Comment:
Same here, maybe `v/verbose` instead of `i/info`?
Usually, `i` is used for `input`, that may be confusing to users.